### PR TITLE
DIG-1529: Integration tests use new synthetic data

### PR DESCRIPTION
**Pull request by mshadbolt**
Should be merged after https://github.com/CanDIG/CanDIGv2/pull/613

- Adds to `ingest_setup.sh` to generate the small_dataset ingest jsons and save them into the `lib/candig-ingest/candigv2-ingest` repo
- Uses the new synthetic data to ingest as part of integration tests
- Modifies several integration tests to work with the new data
- Changes  a few of the tokens from site admin to specific users 
- Adds a new target to makefile to run an individual test and print output whether passing or not `test-integration-%` where % is the name of the test
- Propose deleting running the htsget unit tests
- Propose deleting the add samples to htsget test `test_htsget_add_sample_to_dataset()`

### CODE REVIEW

**Review by OrdiNeu — APPROVED** [review]
Tests ran clean; though I didn't get the chance to dig too deeply into the code, it does look good :heavy_check_mark:

**Comment by mshadbolt** [comment]
Thanks for review @OrdiNeu , will wait on review from @daisieh too due to the changes to htsget tests

**Review by daisieh — APPROVED** [review]
I'm pretty sure this looks fine from an HTSGet point of view; I'll see how sad I am when using my RapidAPI shortcuts without the test_htsget bit running anymore!

**Pull request merged**
